packages/smooth_app/lib/views/bottom_sheet_views/social_handle_view.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:smooth_app/generic_lib/widgets/smooth_list_tile.dart';
+import 'package:smooth_app/helpers/launch_url_helper.dart';
+
+class SocialHandleView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      child: SizedBox(
+        height: MediaQuery.of(context).size.height * 0.8,
+        child: ListView(
+          shrinkWrap: true,
+          scrollDirection: Axis.vertical,
+          children: <Widget>[
+            Container(
+              padding: const EdgeInsets.symmetric(horizontal: 20.0),
+              margin: const EdgeInsets.only(top: 20.0, bottom: 24.0),
+              child: Text(
+                AppLocalizations.of(context)!.connect_with_us,
+                style: Theme.of(context).textTheme.headline1,
+              ),
+            ),
+            // Instagram
+            SmoothListTile(
+                text: AppLocalizations.of(context)!.instagram,
+                leadingWidget: const Icon(Icons.open_in_new),
+                onPressed: () => _launchUrl(
+                    context, 'https://instagram.com/open.food.facts')),
+
+            //Twitter
+            SmoothListTile(
+              text: AppLocalizations.of(context)!.twitter,
+              leadingWidget: const Icon(Icons.open_in_new),
+              onPressed: () =>
+                  _launchUrl(context, 'https://www.twitter.com/openfoodfacts'),
+            ),
+
+            //Blogger
+            SmoothListTile(
+              text: AppLocalizations.of(context)!.blogger,
```

Comment:
Could you change "blogger" by "blog" please?

Reply:
Actually I think Blogger makes sense as there is also 

"Open Food Facts blog" https://en.blog.openfoodfacts.org

Reply:
Sure @g123k

Reply:
> Actually I think Blogger makes sense as there is also
> 
> "Open Food Facts blog" https://en.blog.openfoodfacts.org

I also think blog fits this situation, as blogger is used for someone who writes a blog.

Reply:
Blogger is a trademark. The generic term is Blog